src/restful/controllers/applicationsController.js

```diff
@@ -14,7 +14,7 @@ async function createRoom(db, roomId, user_id, user_name, user_image_url, creato
     id: roomId,
     createdAt: new Date().toString(),
     client: user_id,
-    creator: creator_id,
+    brand: creator_id,
```

Comment:
Clients are brands

So sorry for the confusion lol this is exactly why I felt it needed to be reverted

Reply:
Ahh oh I see, so creator just means like a content creator? Thought you meant creator of the opportunity.

So to clarify. its brands and creators. Brands create opportunities and creators apply to them?

Reply:
Ahh yeah, exactly. It means content creator :) 

That's right!